modules/distr-tracing-tempo-install-instance.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-instance_{context}"]
+= Installing a {TempoName} instance
+
+https://issues.redhat.com/browse/TRACING-3296
+
+Deploying the TempoStack
+
+You can use the OpenShift Container Platform web console to deploy the TempoStack.
+
+.Prerequisites
+
+* Supported Object Store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Install the Tempo Operator:
+
+.. In the OpenShift Container Platform web console, click *Operators* → *OperatorHub*.
+
+.. Choose *Tempo Operator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces* on the cluster.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+IMPORTANT: You must specify the `openshift-operators-redhat` namespace. The `openshift-operators` namespace might contain Community Operators, which are untrusted and might publish a metric with the same name as an OpenShift Container Platform metric, which would cause conflicts.
```

Comment:
@andreasgerstmayr should the namespace be `openshift-tempo-operator`?

Reply:
Yeah, according to our discussion. The namespace should be `openshift-tempo-operator` Also in the installed namespace, there will be two options to select from.

*Operator recommended Namespace: openshift-tempo-operator
Select a namespace.

So this `Under *Installed Namespace*, select *openshift-operators-redhat*.` will be `Under *Installed Namespace*, select *Operator recommended Namespace: openshift-tempo-operator*.`

Reply:
The Tempo operator version has support for monitoring as well so we will have the option to enable monitoring on the namespace. 

`Enable Operator recommended cluster monitoring on this Namespace`

This adds `openshift.io/cluster-monitoring: "true"` label to the namespace and allows OpenShift Monitoring platform to monitor the operator.

Reply:
We also need to add this note under the enable monitoring option.

Namespace monitoring
Please note that installing non-Red Hat operators into OpenShift namespaces and enabling monitoring voids user support. Enabling cluster monitoring for non-Red Hat operators can lead to malicious metrics data overriding existing cluster metrics. For more information, see the [cluster monitoring documentation](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/monitoring/configuring-the-monitoring-stack#maintenance-and-support_configuring-the-monitoring-stack).

Since we do not have the product build yet, you can check an existing operator like Elasticsearch operator which has option to enable monitoring.

Reply:
I've noted some of the comments for discussion as part of the next PR.
Resolving this thread.